src/answers.sql

```diff
@@ -1,15 +1,58 @@
 -- Your answers here:
 -- 1
-
+SELECT c.name, COUNT(s.name) FROM countries c
+    JOIN states s ON c.id = s.country_id
+    GROUP BY c.name;
 -- 2
-
+SELECT COUNT(id) employees_without_bosses FROM employees
+    WHERE supervisor_id IS NULL;
 -- 3
-
+SELECT c.name, o.address, COUNT(e.id) count FROM offices o
+    JOIN countries c ON o.country_id = c.id
+    JOIN employees e ON o.id = e.office_id
+    GROUP BY c.name, o.address
+    ORDER BY count DESC, c.name
+    LIMIT 5;
 -- 4
-
+SELECT supervisor_id, COUNT(supervisor_id) count FROM employees
+    GROUP BY supervisor_id
+    ORDER BY count DESC
+    LIMIT 3;
 -- 5
-
+SELECT COUNT(id) list_of_office FROM offices
+    WHERE state_id = (
+        SELECT id FROM states WHERE name = 'Colorado'
+    );
```

Comment:
i think we could we add a validation for country too

Reply:
Done.